index.html

```diff
@@ -7,23 +7,30 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
   </head>
 
   <body>
-
-      <header class="toolbar">
+    <section class="section-1">
+      <nav class="toolbar">
         
         <div class="toolbar-txt">My Logo</div>
-
+        <a href="#" class="toggle-button">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </a>
+        
         <div class="item-nav">
-          <div class="item-txt">Portifolio</div>
-          <div class="item-txt">About</div>
-          <div class="item-txt">Contact</div>
+          <button class="toggle-2">X</button>
+          <ul>
+            <li><a href="#work" onclick="closeMenu()">Portfolio</a></li>
+            <li><a href="#about-myself" onclick="closeMenu()">About</a></li>
+            <li><a href="#contact-form" onclick="closeMenu()">Contact</a></li>
+          </ul>
         </div>
-        
-        <button class="btn"><i class="bars"></i></button>
```

Comment:
- [ ] Kindly add back the close menu icon and its functionality to close the menu popup, it would make your design look like the original, just as the previous reviewer requested the close button should only be removed from the desktop version 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_568](https://user-images.githubusercontent.com/69638013/208677861-1bba88a4-64e4-4cf6-8893-956efdaf4139.png)  |  ![Screenshot_567](https://user-images.githubusercontent.com/69638013/208677700-2f263301-8704-4689-9584-6069efb4a9ca.png)


### OPTIONAL

- [ ] It would be better for the background not to be scrollable when the mobile nav is open, this will create a better user experience  👍🏿